src/nyc_trees/js/src/surveyPage.js

```diff
@@ -458,8 +458,12 @@ function submitSurveyWithTrees() {
             data: JSON.stringify(data)
         })
         .done(function(content) {
+            // make sure not to take the hashstring along for the ride
+            var href = window.location.origin +
+                    window.location.pathname +
+                    'confirm/' + content.survey_id + '/';
```

Comment:
Whenever we embed paths in JS, we have been leaving comments for future refactorers in the js and py files that say something like "this URL needs to be kept in sync with such/and/such/view".